skedge-backend/database/services/endpoint-service.js

```diff
@@ -9,7 +9,11 @@ module.exports = {
         return database_services.getCourseCatalog();
     },
 
-    createUser: function () {
-      return database_services.createUser();
+    createUser: function (objectJS) {
```

Comment:
Inconsistent names (objectJS, objectJSON). How about a name that is more indicative of what the object contains?